services/gcp/iap/tunnelDestGroups.yml

```diff
@@ -0,0 +1,50 @@
+name: Identity Aware Proxy tunnel destination groups resource type.
+description: >-
+  Refers to a particular group of VMs in you project.  VMs are collected in a list of either CIDRs or FQDNs.
+scope: HIGH
+notes: >-
+  IAP is used to control access to cloud services.  Changes to IAP related settings could remove access from mission-critical applications or 
+  grant an attacker access to sensitive resources.
+privileges:
+  accessViaIAP:
+    risks: [impact:dos, escalation:network]
```

Comment:
I am looking at this.  My thinking is that since `accessViaIAP` grants a principal access to an app, removing that would create a dos? 

>The iap.webServiceVersions.accessViaIAP permission grants user access to an app.

<img width="894" alt="Screenshot 2024-08-21 at 2 35 34 PM" src="https://github.com/user-attachments/assets/5966a130-c181-4b9e-a988-0c5dc6fbeed6">

https://cloud.google.com/iap/docs/managing-access#roles